corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -186,6 +186,13 @@ def get_values_for_query(self, query_context):
     def get_choices_for_known_values(self, values, user):
         return []
 
+    def _make_choice_from_value(self, value):
+        if value is None:
+            return Choice(NONE_CHOICE, '[Missing]')
+        elif not value:
+            return Choice(EMPTY_CHOICE, '[Empty]')
```

Comment:
If these values aren't sent then `select2` appears to just throw them away, which is why I couldn't just use an empty string here.